src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -172,6 +186,7 @@ public static Test createTest(long startTime, CommandLine commandline, Options o
         final int eventsPerSec;
         final int eventsPerProducer;
         final int eventsPerConsumer;
+        final int flushEventsPerProducer;
```

Comment:
In what sense is this "perProducer"? Isn't it just a configuration parameter for the writer?

Reply:
yes.. but , since there is variable named  eventsPerProducer , so  i thought of keeping in the same fashion.

Reply:
Right, but that one makes sense. IE if there are 100 events per producer and 10 producers there are 1000 total events.

If there are 10 producers and eventsPerFlush is 10... Then nothing. There is no way to multiply or divide those that gives a number that means anything.

Reply:
yes your right.. its up to the user/Performance engineer should take care of it. This is introduced because performance team and Raul are having IO issues in PKS; This option was required to observe the performance variations by flushing the events. 
dividing among multiple producers may create more confusions.